### ISSUE TITLE
Is it possible to run this in Xbox Companion App ?

### ISSUE THREAD

**Issue by matejbulik** [fqlx/XboxKeyboardMouse#207]
Xbox beta app doesn't stream gemes anymore, you can use only Xbox companion app. Is it possible to run the program with it? My keyboard works well with the new app, but mouse just not lock and can't be used. If I start the older Xbox app, mouse lock to it and works well.

**Comment by kiwioGIT** [comment]
Yes, i have that problem too. It is becouse the xbox companion app is not called xbox and the program checks if the app named "xbox" is running in foreground. Is there a way to download older versions of xbox app where you were able to stream to console?
